grails-app/controllers/imod/ImodController.groovy

```diff
@@ -67,7 +68,7 @@ class ImodController {
                 lastName: 'Bansal',
                 email: 'ajay.bansal@asu.edu',
                 role: 'Professor',
-                officeHours: 'Tuesdays & Thursdays | 9:00am – 10:0am or by appointment',
+                officeHours: 'Tuesdays & Thursdays | 9:00am â€“ 10:0am or by appointment',

```

Comment:
@ChristianMurphy  I didn't do any of those changes. I suppose these are because of the GGTS Editor error i am getting.